modules/packages/SortedSet/Treap.chpl

```diff
@@ -64,7 +64,7 @@ module Treap {
   }
 
   @chpldoc.nodoc
-  var _treapRandomStream = new RandomStream(int);
+  var _treapRandomStream = new randomStream(int, true);
```

Comment:
I assume this is requesting a `parSafe` randomStream? But I don't see how this could compile with `record randomStream` as defined. Maybe the Treap could use its own lock for its own random stream?

Reply:
I think it might be passing in `true` for the `seed`?

Reply:
You're probably right it took `true` for the seed. I'll work on adding a lock to Treap to maintain the current behavior.

Reply:
Actually, it looks like the data structure already locks around calls to `_Random`.